src/components/buttonMore/button-more.js

```diff
@@ -0,0 +1,49 @@
+import template from "./button-more.html";
+import { MDCMenu } from "@material/menu";
+
+export class ButtonMoreComponent {
+  constructor(mountPoint, position) {
+    this.mountPoint = mountPoint;
+    this.position = position;
+  }
+
+  querySelectors() {
+    this.menuRoot = this.mountPoint.querySelector(".menu");
+    this.buttonMore = this.mountPoint.querySelector(".button-more__more");
+  }
+
+  initPosition() {
+    switch (this.position) {
+      case "left":
+        this.menuRoot.classList.add("button-more__popup-left");
+        break;
+      case "right":
+        this.menuRoot.classList.add("button-more__popup-right");
+        break;
+    }
+  }
+
+  initMDC() {
+    this.menu = new MDCMenu(this.menuRoot);
+  }
+
+  addEventListener() {
+    this.buttonMore.addEventListener("click", this.handleBtnClick.bind(this));
+  }
```

Comment:
I do not see handlers for `edit` or `delete` click -- please add these and expose them via `props` -- so parent component will be able to handle it. For now you can `console.log` smth in each handler